### DISCUSSION TITLE
Local commits not showing on Github

### DISCUSSION THREAD

**Discussion by gswee** [community/community#21844] [New to GitHub]
<p>Hello,</p>
<p>I’m working on a Godot project and I’m using Git integration to get used to it.</p>
<p>However, I have run into a dilemma where my local commits in Git GUI are not showing up on Github.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/9eaa3a6e86ba5d3a4b908ae3b4760258e5ff0ce1.png" data-download-href="/uploads/short-url/mDC9sS6yuvwhSYtCSPNvQWiVGHT.png?dl=1" title="Annotation 2020-01-26 190733.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/18357809/181067102-3a351215-06ec-4793-b866-295bfd8affd9.png" alt="Annotation 2020-01-26 190733.png" data-base62-sha1="mDC9sS6yuvwhSYtCSPNvQWiVGHT" width="690" height="67" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/9eaa3a6e86ba5d3a4b908ae3b4760258e5ff0ce1_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Annotation 2020-01-26 190733.png</span><span class="informations">809×79 4.27 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div> <img src="https://user-images.githubusercontent.com/18357809/181067104-fbec8668-99d2-46d0-aae5-ab5670cd0fdb.png" alt="Annotation 2020-01-26 190936.png" data-base62-sha1="dOeEOsYFgBA44KBNN5dPYzoNSvN" width="349" height="317">
<p>Is it because the author name in 1st pic is different even though email is the same? Do let me know any other details that would help.</p>

**Comment by FranklinYu** [comment, marked as answer]
<p>You need to push them to GitHub before it can be shown online.</p>

**Comment by gswee** [comment]
<p>Ah, so I have to Push before I Commit…I swear I forgot that step back in my uni study</p>
<p>Thank you so much!</p>
